setup.py

```diff
@@ -46,6 +46,7 @@
     'pytz>=2018.04',
     'pyyaml>=3.13,<4',
     'requests==2.19.1',
+    'ruamel.yaml==0.15.72',
```

Comment:
FYI this is one of the very rare occasions that we add a core dependency. And it's only because ruamel will eventually replace pyyaml.